task3.js

```diff
@@ -0,0 +1,15 @@
+var a = +prompt('введите переменную а');
+var b = +prompt('введите переменную b');
+
+function mathematicalOperation() {
+
+	if (a >= 0 && b >= 0) {
+		return (a - b);
+	} else if (a < 0 && b < 0) {
+		return (a * b);
+	} else if ((a >= 0 && b < 0) || (a < 0 && b >= 0)) {
```

Comment:
Здесь можно просто else - результат будет тот же
```suggestion
	} else {
```